api/migrations/2024101114300_uk-stations.cjs

```diff
@@ -0,0 +1,32 @@
+/**
+ * Template for
+ * {@link https://salsita.github.io/node-pg-migrate/#/migrations?id=defining-migrations defining migrations}.
+ *
+ * @type {{
+ *   down: (pgm: import("node-pg-migrate").MigrationBuilder) => void | Promise<void>;
+ *   shorthands: Record<string, import("node-pg-migrate").ColumnDefinition>;
+ *   up: (pgm: import("node-pg-migrate").MigrationBuilder) => void | Promise<void>;
+ * }}
+ */
+const migration = {
+	up(pgm) {
+		pgm.createTable("uk_stations", {
+			station_id: { notNull: true, primaryKey: true, type: "int8" },
+			station_name: { notNull: true, type: "text" },
+			sixteen_character_name: { notNull: true, type: "text" },
+			longitude: { notNull: true, type: "float8" },
+			latitude: { notNull: true, type: "float8" },
+			crs_code: { notNull: true, type: "text" },
```

Comment:
Maybe too fussy for now, but we know that crs_code is char(3).  We should perhaps reflect this in the column type

Reply:
Yes, it is better to set the working environment more stable for manual mistakes. Changed the type to char(3)